builtin/providers/fastly/resource_fastly_service_v1.go

```diff
@@ -188,7 +188,19 @@ func resourceServiceV1() *schema.Resource {
 							Type:        schema.TypeString,
 							Optional:    true,
 							Default:     "",
-							Description: "SSL certificate hostname",
+							Description: "SSL certificate hostname (deprecated by Fastly)",
```

Comment:
If this is deprecated in fastly, then do you think we should also deprecate this in Terraform?

If so, helper schema has the following:

Deprecated: "<message goes here>"

Please be aware that this will have to be a backwards compatible change that we accept both values for a while and make the code work